machine.go

```diff
@@ -352,12 +353,47 @@ func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper) error {
 		"modules.builtin.bin",
 		"modules.devname"}
 
+	moddir := "/lib/modules"
+	if mergedUsrSystem() {
+		moddir = "/usr/lib/modules"
+	}
+
+	// build a list of built-in modules so that we don’t attempt to copy them
+	var builtinModules = make(map[string]bool)
+
+	f, err := os.Open(path.Join(moddir, kernelRelease, "modules.builtin"))
+	if err != nil {
```

Comment:
hm, if the file does not exist, we should not attempt to load `modules.builtin` and just load the modules normally ?
what about systems that use `modules.builtin.bin` ?

Reply:
I was considering this, but then I realised we already unconditionally copy `modules.builtin` (unless I’m misreading the code), so it is already required to be present.